yarn.lock

```diff
@@ -12496,6 +12496,18 @@ loose-envify@^1.1.0, loose-envify@^1.4.0:
   dependencies:
     js-tokens "^3.0.0 || ^4.0.0"
 
+lottie-react@^2.3.1:
```

Comment:
you need to run `yarn` again and the lock file should be updated with this new dependency version without `^`